src/components/Profile/ProfileInfo/Status.tsx

```diff
@@ -23,7 +23,7 @@ const Status: FC = () => {
     dispatch(updateStatus(profileStatus));
   };
 
-  const onStatusChange = (e: any) => {
+  const onStatusChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
no need to import all react types every time. you can use it as `React.FC`, `React.ChangeEvent`